### PR TITLE
Refactor LogitNormal

### PR DESCRIPTION
**Pull request by themrzmaster**
partially helps #4686

### CODE REVIEW

File: pymc3/tests/test_distributions_random.py [hunk 1]
```diff
@@ -513,6 +509,22 @@ class TestNormal(BaseTestDistribution):
         "check_pymc_draws_match_reference",
         "check_rv_size",
     ]
+class TestLogitNormal(BaseTestDistribution):
+    def logit_normal_rng_fn(self, rng, size, loc, scale):
+        return expit(st.norm.rvs(loc=loc, scale=scale, size=size, random_state=rng))
```
Comment (themrzmaster) [inline pymc3/tests/test_distributions_random.py:514]: is this the best way to test? I am having a problem here

File: pymc3/distributions/continuous.py [hunk 2]
```diff
@@ -3726,6 +3726,18 @@ def logcdf(value, mu, s):
...
+class LogitNormalRV(RandomVariable):
+    name = "logit_normal"
+    ndim_supp = 0
+    ndims_params = [0, 0]
+    dtype = "floatX"
+    _print_name = ("logitNormal", "\\operatorname{logitNormal}")
+
+    @classmethod
+    def rng_fn(cls, rng, mu, sigma, size=None):
+        return invlogit(stats.norm.rvs(loc=mu, scale=sigma, size=size, random_state=rng))
```
Comment (ricardoV94) [inline pymc3/distributions/continuous.py]: Should use `scipy.expit`. I think `invlogit` is the aesara symbolic counterpart. Here the output it shouldn't be symbolic anymore

File: pymc3/tests/test_distributions_random.py (hunk 1 above)
Comment (ricardoV94) [reply pymc3/tests/test_distributions_random.py:514]: What problem are you having? Maybe it's related to my comment above?

**Comment by ricardoV94** [comment]
@themrzmaster Can you add a test for the tau parametrization in test_distributions_random? There are some examples with the normal that you can copy from.

**Review by ricardoV94 — APPROVED** [review]
LGTM. Thank you for opening the PR!

**Pull request merged**